physionet-django/console/views.py

```diff
@@ -1403,6 +1403,17 @@ def credential_processing(request):
     final_applications = applications.filter(
         credential_review__status=60).order_by('application_datetime')
 
+    if request.method == 'POST':
+        if 'reset_application' in request.POST:
+            try:
+                application = CredentialApplication.objects.get(slug=request.POST['reset_application'])
+            except CredentialApplication.DoesNotExist:
+                raise Http404()
+            application.credential_review.delete()
```

Comment:
Perhaps put these database actions (delete and create) into a `with transaction.atomic():` clause so that if one step fails, both fail?